common/src/main/java/net/william278/husksync/config/Settings.java

```diff
@@ -100,26 +100,12 @@ public static class DatabaseCredentials {
             private String database = "HuskSync";
             private String username = "root";
             private String password = "pa55w0rd";
+            private String authDb = "admin";
```

Comment:
can we call this `mongoAuthDb`, move it below the `parameters`, and add a `@Comment`?